torch/quantization/fuse_modules.py

```diff
@@ -0,0 +1,131 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import torch
+import copy
+
+import torch.nn._intrinsic.modules.fused as torch_fused
+
+def fuse_conv_bn(conv, bn):
+    r"""Given the conv and bn modules, fuses them and returns the fused module
+
+    Args:
+        conv: Module instance of type conv2d
+        bn: Spatial BN instance that needs to be fused with the conv
+
+    Examples::
+
+        >>> m1 = nn.Conv2d(10, 20, 3)
+        >>> b1 = nn.BatchNorm2d(20)
+        >>> m2 = fuse_conv_bn(m1, b1)
+    """
+    assert(conv.training == bn.training),\
+        "Conv and BN both must be in the same mode (train or eval)."
+
+    if conv.training:
+        return torch.nn._intrinsic.ConvBn2d(conv, bn)
+    else:
+        fused_conv = copy.deepcopy(conv)
```

Comment:
Please re-factor the code from line 27 to 46 as a seperate function fuse_conv_bn_inference and move that code to nn/utils.